### ISSUE TITLE
Windows Phone Dialog issue

### ISSUE THREAD

**Issue by oste** [angular/material#5418]
I am doing some browser testing using browserstack.com and I found that the Nokia Windows Phone models 930, 630, 520 have a dialog issue. When you scroll down the page some and then open up the dialog it will be cut off. It is interesting to note that the Nokia 925 does not have this problem. I am going to try to isolate this issue, but wanted to get this information up for now.

**Comment by oste** [comment]
This issue is not present in version 0.9.8. As soon as I update to V0.10 the modal will be forced to the top of the page as seen in the screenshot.

**Comment by EladBezalel** [comment]
You should update to HEAD or at least 1.0-rc2 and check again, if this issue returns please write here and i will open this issue again and will investigate it.

**Comment by oste** [comment]
@EladBezalel this issue is not resolved in 1.0-rc2. Here is a demo link using rc1 I can't use rc2 b/c the rc2 assets are giving a 404. 

https://s3-us-west-2.amazonaws.com/oste-static/index.html. 

I tested on rc2 as well and the issue exists. Here is a pen http://codepen.io/anon/pen/Vvdzgr but strange enough this actually works on windows phone. So I hosted that s3 site for you, I can update that to rc2 as soon as those assets are available.

@ThomasBurleson I am hoping that this can make it into the V1 roadmap. The dialog is a very important component and it is unusable on Windows phone currently.
